### DISCUSSION TITLE
折腾中国移动家用宽带

### DISCUSSION THREAD

**Discussion by giscus** [liolok/liolok.com#5] [Comments]
# 折腾中国移动家用宽带

https://liolok.com/zhs/customize-china-mobile-home-broadband/

**Comment by WeepingDogel** [comment]
偷偷坐沙发～.
.

**Comment by iamydp** [comment]
江苏移动只要重拨v6前缀都会变，之前有想过拨号时候使用静态v6固定下来，虽然成功了，但万一运营商前缀一变所有依赖v6的服务都挂，想想还是native+ddns稳一点吧。现在手机开个流量都有v6地址，在外维护也挺方便的。最主要的是和朋友联机，晚高峰中日延迟只有80多毫秒，测速能把我的上行吃满，也算很好了。

**Reply by liolok** [reply]
我这里只用来挂 PT 没什么服务，所以动态也就动态了。只是听说现在移动都是「随机下发」不能直接改光猫配置了，真难啊。
